stable/helm-exporter/values.yaml

```diff
@@ -26,3 +26,10 @@ serviceAccount:
   # The name of the ServiceAccount to use.
   # If not set and create is true, a name is generated using the fullname template
   name:
+
+serviceMonitor:
+  # Specifies whether a ServiceMonitor should be created
+  create: false
+  interval: 30s
```

Comment:
Lets also leave this as empty and only set it if the user specifies it.  This will override the default that comes from the prometheus operator if I'm not mistaken.

Reply:
Those keys are expected in templates/servicemonitor.yaml. What should be empty exactly?

Reply:
`interval` so it inherits the default from the prometheus-operator.  Or do we want to force it to be 30s?

Reply:
I've checked other charts which provide service monitor and all of them define some default interval.
I think using the default interval coming from operator is a bit too clever but we can do this if you insist.

Reply:
Since the interval is not a required field unless we have a good reason to think everyone wants to or should be scrapping at 30s I would prefer to default it to false.  That way it uses the defaults instead of being overriden.